drivers/wdf/wdfldr/ntddk_ex.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include <ntdef.h>
+
+typedef enum _SYSTEM_INFORMATION_CLASS {
```

Comment:
This file should not be needed, structures are defined in NDK. Maybe some of them are missing, then they should be added there